test/library/packages/Python/customType.chpl

```diff
@@ -0,0 +1,45 @@
+use Python;
+import Reflection;
+use CTypes;
+
+record myRec {
+  var x: int;
+  var y: string;
+}
+
+class myRecConverter: TypeConverter {
+  var pyClsType: borrowed Class;
+  override proc handlesType(type T): bool {
+    return T == myRec;
+  }
+  override proc toPython(interpreter: borrowed Interpreter,
+                         type T, const value: T): Python.CPythonInterface.PyObjectPtr throws {
+    if T != myRec then halt("Expected myRec");
+    var pyObj = owned.release(pyClsType(value.x, value.y));
+    pyObj.obj!.isOwned = false;
+    var ptr = pyObj.obj!.get();
+    delete pyObj;
```

Comment:
Consider defining a `PyObject.release` type method that does this, so that you can do:

```Chapel
return PyObject.release(myClsType(value.x, value.y))
```

Since this is part of the user-facing API (that's how you expect users to define their own converters), I think dealing with the internals of `PyObject` and its `isOwned` field is not ideal.